languages.yaml

```
@@ -2255,25 +2255,32 @@ Toml:
 Tsx:
   type: programming
   ascii: |
-    {0}          .REACT.    .REACT.
-    {0}        TS     .TSXTSX.     TS
-    {0}       TS       :TSTS:       TS
-    {0}       TS     .REACTTSX'     TS
-    {0}     'TSXTSXTSXTSXTSXTSXTSXTSXTS'
-    {0}  'REACTTSREACT        REACTTSREACT'
-    {0} 'TSX'  REACT'  :TSXJ:  'REACT  'TSX'
-    {0}'TS      TSX'  :TSXTSX:  'TSX      TS'
-    {0}'TS      TSX'  :TSXTSX:  'TSX      TS'
-    {0} 'TSX'  REACT'  :TSXJ:  'REACT  'TSX'
-    {0}  'REACTTSREACT        REACTTSREACT'
-    {0}     'TSXTSXTSXTSXTSXTSXTSXTSXTS'
-    {0}       TS     .REACTTSX'     TS
-    {0}       TS       :TSTS:       TS
-    {0}        TS     .TSXTSX.     TS
-    {0}          .REACT.    .REACT.
+    {0}TSXTSXTSXTSXTSXTSXTSXTSXTSXTSXTSXTSX{1}TSX
+    {0}TSXTSXTSXTSXTSXTSXTSXTSXTSXTSXTSXTS{1}XTSX
+    {0}TSXTSXTSXTSXTSXTSXTSXTSXTSXTSXTSXT{1}SXTSX
+    {0}TSXTSXTSXTSXTSXTSXTSXTSXTSXTSXTSX{1}TSXTSX
+    {0}TSXTSXTSXTSXTSXTSXTSXTSXTSXTSXST{1}SXTSXTS
+    {0}TSXTSXTSXTSXTSXTSXTSXTSXTSXTSXT{1}SXTSXTSX
+    {0}TSXTSXTSXTSXTSXTSXTSXTSXTSXTSX{1}TSXTSXTSX
+    {0}TSXTSXTSXTSXTSXTSXTSXTSXTSXTS{1}XTSXTSXTSX
+    {0}TSXTSXTSXTSXTSXTSXTSXTSXTSXT{1}SXTSXTSXTSX
+    {0}TSXTSXTSXTSXTSXTSXTSXTSXTSX{1}TSXTSXTSXTSX
+    {0}TS        SX      TSXTSXTS{1}X{2}TSX{1}TSX{2}TSX{1}TSX
+    {0}TSXTS  SXTS   XTSXTSXSTSX{1}TSX{2}TSX{1}T{2}SXT{1}SXTS
+    {0}TSXTS  SXTSXT   SXTSXTSX{1}TSXTSX{2}TSX{1}TSXTSX
+    {0}TSXTS  SXTSXTSX   TSXTS{1}XTSXTS{2}XTSXT{1}SXTSX
+    {0}TSXTS  SXTS   XT   SXT{1}SXTSXT{2}SXT{1}S{2}XTS{1}XTSX
+    {0}TSXTS  SXTSX      TSX{1}TSXTSX{2}TSX{1}TSX{2}TSX{1}TSX
+    {0}TSXTSXTSXTSXTSXTSXTS{1}XTSXTSXTSXTSXTSXTSX
```

```suggestion
    {0}TSXTSXTSXTSXTSXTSXTSXTSXTSXTSXTSXTSX{1}TSX
    {0}TSXTSXTSXTSXTSXTSXTSXTSXTSXTSXTSXTS{1}XTSX
    {0}TSXTSXTSXTSXTSXTSXTSXTSXTSXTSXTSXT{1}SXTSX
    {0}TSXTSXTSXTSXTSXTSXTSXTSXTSXTSXTSX{1}TSXTSX
    {0}TSXTSXTSXTSXTSXTSXTSXTSXTSXTSXST{1}SXTSXTS
    {0}TSXTSXTSXTSXTSXTSXTSXTSXTSXTSXT{1}SXTSXTSX
    {0}TSXTSXTSXTSXTSXTSXTSXTSXTSXTSX{1}TSXTSXTSX
    {0}TSXTSXTSXTSXTSXTSXTSXTSXTSXTS{1}XTSXTSXTSX
    {0}TSXTSXTSXTSXTSXTSXTSXTSXTSXT{1}SXTSXTSXTSX
    {0}TSXTSXTSXTSXTSXTSXTSXTSXTSX{1}TSXTSXTSXTSX
    {0}TS        XT      SXTSXTSX{1}T{2}SXT{1}SXT{2}SXT{1}SXT
    {0}TSXTS  XTSX   TSXTSXTSXTS{1}XTS{2}XTS{1}X{2}TSX{1}TSXT
    {0}TSXTS  XTSXTS   XTSXTSXT{1}SXTSXT{2}SXT{1}SXTSXT
    {0}TSXTS  XTSXTSXT   SXTSX{1}TSXTSX{2}TSXTS{1}XTSXT
    {0}TSXTS  XTSX   TS   XTS{1}XTSXTS{2}XTS{1}X{2}TSX{1}TSXT
    {0}TSXTS  XTSXT      SXT{1}SXTSXT{2}SXT{1}SXT{2}SXT{1}SXT
    {0}TSXTSXTSXTSXTSXTSXTS{1}XTSXTSXTSXTSXTSXTSX
```

fix repeating sequence

---

Pattern fixed, thanks :+1: